tests/mock_tables/mock_multi_asic.py

```diff
@@ -14,7 +13,10 @@ def mock_is_multi_asic():
 def mock_get_namespace_list(namespace=None):
     return ['asic0', 'asic1']
 
+def mock_get_namespace_for_port(port):
```

Comment:
Why to we need to mock get_namespace_for_port. I think if we mock the get_namespace_list, get_namespace_for_port API should work. Any issues seen ?

Reply:
I have removed the same , this mock is not needed.